### DISCUSSION TITLE
Tritanopia Colorblind Theme missing dark mode for projects beta

### DISCUSSION THREAD

**Discussion by tattan** [community/community#15200] [Accessibility] [answered]
I'm assuming this is still in the works but for now, the best dark mode for me is still the normal dark mode until this gets added. 

I'm looking forward to it though as the old projects and issues are much more distinguishable for me!

**Comment by cehfisher** [comment, marked as answer]
Hello @tattan - thank you for providing this feedback! After reviewing your issue report against the latest UI, this issue appears to be resolved now. Could you please confirm and close this issue? Thank you!

**Comment by tattan** [comment]
Yes, I'm currently using the theme!
